crates/turborepo-micro-frontend/src/error.rs

```diff
@@ -0,0 +1,26 @@
+use std::fmt;
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error("Unable to read configuration file: {0}")]
+    Io(#[from] std::io::Error),
+    #[error("Unable to parse JSON: {0}")]
+    JsonParse(String),
+}
+
+impl Error {
+    /// Constructs an error message from multiple biome diagnostic errors
+    pub fn biome_error(errors: Vec<biome_diagnostics::Error>) -> Self {
```

Comment:
we already have a lot of this logic in `turborepo-errors/lib.rs` if you want to pull from that